Python/Product/PythonTools/PythonTools/Project/PythonProjectNode.cs

```diff
@@ -2161,8 +2161,13 @@ bool addNew
             var path = factory.Configuration.PrefixPath;
             if (removeFromStorage && Directory.Exists(path)) {
                 var t = Task.Run(() => {
-                    Directory.Delete(path, true);
-                    return true;
+                    try {
+                        Directory.Delete(path, true);

```

Comment:
The `return false` here is already used to trigger a friendly message rather than the generic "See ActivityLog.xml" message.